Framework/Algorithms/src/MonteCarloAbsorption.cpp

```diff
@@ -184,381 +206,57 @@ void MonteCarloAbsorption::exec() {
       } else {
         // elastic case already initialized
       }
-      doSimulation(detPos, lambdaIn, lambdaOut, yValues[bin], eValues[bin]);
+      std::tie(signal[j], std::ignore) =
```

Comment:
I don't have any well informed opinion I'm afraid. Yes, the other algorithms produce all-zero errors so I'd also go for consistency.
Maybe this could be brought up to relevant expert users. Is the fact that the errors are usually set to zero a feature/well justified, or is it a limitation/improvable? Are they ever used in any other calculations?